src/components/Thanks/SingleVersion/OptInModule.vue

```diff
@@ -142,6 +144,10 @@ const updateOptIn = async value => {
 		} else {
 			await updateCommunicationSettings(value, value, false);
 		}
+		setMailUpdatesOptOutCookie(false);
+	} else {
+		const loanId = props.loans[0]?.id || null;
+		setMailUpdatesOptOutCookie(true, loanId);
```

Comment:
We instead want to set this cookie when the TY page loads, so we know which loan to show on the new card in MyKiva. These changes are requiring a user to interact with the opt-in module on the TY page.

Reply:
We can get last loan from `transactions` query w/o dealing with cookies.

Reply:
True, but we only want to show the new marketing card on MyKiva during the same session as a transaction.